packages/expo-barcode-scanner/src/BarCodeScanner.tsx

```diff
@@ -48,7 +48,7 @@ export { PermissionResponse, PermissionStatus };
 export interface BarCodeScannerProps extends ViewProps {
   type?: 'front' | 'back' | number;
   barCodeTypes?: string[];
-  onBarCodeScanned: BarCodeScannedCallback;
+  onBarCodeScanned: BarCodeScannedCallback | undefined;
```

Comment:
```suggestion
  onBarCodeScanned?: BarCodeScannedCallback;
```

this is equivalent and maybe reads better / is more consistent with other props

example: https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4BYAKCpgE8wk4AhFKAYQgBMkBlNFAHYCknNigA24gEboA1nAC8cABQBKRQD44AbwC+FapToM4ABRxgAzop1U49uBAEt2XXvyEiA-AC5mrDm4+QWFRCWk5A10qKkwAVwEMYCd-VyCPYShlbUdnALdgz044XT9zCCt1bTsHIhg4qAE4AB5OYAA3OAB6DSpow2aXQPcQpChcoYKMkQVtBO5MYFDdbo0gA